### PR TITLE
Add more portuguese channels

### PR DESCRIPTION
**Pull request by edersonferreira**
## Summary:
Add more portuguese channels

Added:
   - [Código Fonte TV](https://www.youtube.com/user/codigofontetv)
   - [Rocketseat](https://www.youtube.com/channel/UCSfwM5u0Kce6Cce8_S72olg)
   - [Filipe Deschamps](https://www.youtube.com/channel/UCU5JicSrEM5A63jkJ2QvGYw)
   - [Fabio Akita](https://www.youtube.com/user/AkitaOnRails)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -204,6 +207,10 @@ vous trouverez sur cette plateform des cours et examens qui vous accompagnerons
 * Portuguese
    * [Loiane Groner](https://www.youtube.com/user/Loianeg)
    * [Curso em Vídeo](https://www.youtube.com/user/cursosemvideo)
+   * [Código Fonte TV](https://www.youtube.com/user/codigofontetv)
+   * [Rockeatseat](https://www.youtube.com/channel/UCSfwM5u0Kce6Cce8_S72olg)
```
Comment (Fcmam5) [inline README.md:211]:
```suggestion
   * [Rocketseat](https://www.youtube.com/channel/UCSfwM5u0Kce6Cce8_S72olg)
```

File: README.md [hunk 2]
```diff
@@ -204,6 +207,10 @@ vous trouverez sur cette plateform des cours et examens qui vous accompagnerons
...
+   * [Felipe Deschamps](https://www.youtube.com/channel/UCU5JicSrEM5A63jkJ2QvGYw)
```
Comment (Fcmam5) [inline README.md:212]:
```suggestion
   * [Filipe Deschamps](https://www.youtube.com/channel/UCU5JicSrEM5A63jkJ2QvGYw)
```

**Review by Fcmam5 — COMMENTED** [review]
Thank you for your contribution :) Just check these minor fixes

**Comment by edersonferreira** [comment]
Thanks to the minor fixes hahaha.

**Comment by edersonferreira** [comment]
Thanks to the minor fixes hahaha.

**Review by Fcmam5 — APPROVED** [review]
Thank you!

**Pull request merged**